### ISSUE TITLE
Surface Web Discovery Project (WDP) opt-in in Rewards onboarding

### ISSUE THREAD

**Issue by Miyayes** [brave/brave-browser#40570]
## Description

In Rewards onboarding, add a slide for WDP opt-in. 

See Designs in Rewards 3.0 Figma.

**Comment by GeetaSarvadnya** [comment]
Verification INPROGRESS on

```
Brave | 1.72.83 Chromium: 130.0.6723.58 (Official Build) beta (64-bit)
-- | --
Revision | b49615851cd2dc4b1c3f5f13d3207d3c032bfd8d
OS | Windows 10 Version 22H2 (Build 19045.5011)
```
- Logged https://github.com/brave/brave-browser/issues/41750
- Asked a clarification via https://bravesoftware.slack.com/archives/CNH9Q6REV/p1729498325319069?thread_ts=1721233592.471079&cid=CNH9Q6REV

**Comment by LaurenWags** [comment]
Changed to `release-notes/exclude` as this will be included in the notes when Rewards 3.0 is released per above and discussion with @Miyayes 👍🏻
